lib/Mojo/Util.pm

```diff
@@ -958,6 +981,21 @@ Unindent multi-line string.
 
 Unquote string.
 
+=head2 urandom_bytes
+
+  my $bytes = urandom_bytes(32);
+
+Returns random bytes. On Linux and UNIX-like systems /dev/urandom is used as a
+source of random data, on Windows RtlGenRandom/SystemFunction036 is used. The
+default number of bytes returned is 32 bytes.
+
+=head2 urandom_urlsafe
+
+  my $token = urandom_urlsafe;
+
+Generates a base64url encoded string of 32 random bytes suitable for session
+tokens and similar.
```

Comment:
The signatures/examples of these functions could show that *both* take an optional argument. Here, it looks like `urandom_bytes()` always needs an argument, and that `urandom_urlsafe()` could never receive an argument.

Reply:
Fixed